README.md

```diff
@@ -1 +1,97 @@
-# RL-based-Maze-Solver
\ No newline at end of file
+# RL-based Maze Solver
+Open Projects 2021
+
+![Input Maze Solution](https://github.com/tushdon2/RL-based-Maze-Solver/blob/main/Images%20and%20Videos/Images/Input%20maze1%20and%20solution.png)
+<br>fig1 <i>Input Maze Solution</i>

```

Comment:
Center align all the images using the `<p>` tag and align attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

Reply:
centred all the images